src/main/java/org/folio/circulation/domain/Item.java

```diff
@@ -22,13 +22,19 @@
 import java.util.UUID;
 import java.util.stream.Collectors;
 
+import org.apache.commons.lang3.StringUtils;
 import org.folio.circulation.domain.representations.ItemProperties;
 import org.folio.circulation.support.JsonArrayHelper;
 
 import io.vertx.core.json.JsonArray;
 import io.vertx.core.json.JsonObject;
 
 public class Item {
+  private static final String CALL_NUMBER_KEY = "callNumber";
```

Comment:
Why are these defined here rather than in `ItemProperties`?

Reply:
done